federator/src/lib/Heartbeat.js

```diff
@@ -150,7 +150,6 @@ module.exports = class Heartbeat {
 
         try {
             for(let log of logs) {
-                this.logger.info('Processing Heartbeat event log:', log);
```

Comment:
we're removing this because the next log already lets us know that we're processing a heartbeat event log, right?

Reply:
exactly 💯